packages/qdrant-loader/src/qdrant_loader/__init__.py

```
@@ -43,7 +43,12 @@ def __getattr__(name):
 
         return QdrantManager
     else:
-        raise AttributeError(f"module '{__name__}' has no attribute '{name}'")
+        try:
+            import importlib
+
+            return importlib.import_module(f"{__name__}.{name}")
+        except (ImportError, ModuleNotFoundError):
+            raise AttributeError(f"module '{__name__}' has no attribute '{name}'")
```

Ruff (0.15.12)

[warning] Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)